dbos/_sys_db.py

```diff
@@ -692,6 +700,45 @@ def get_workflows(self, input: GetWorkflowsInput) -> GetWorkflowsOutput:
 
         return GetWorkflowsOutput(workflow_uuids)
 
+    def get_queued_workflows(
+        self, input: GetQueuedWorkflowsInput
+    ) -> GetWorkflowsOutput:
+
+        query = sa.select(SystemSchema.workflow_queue.c.workflow_uuid).join(
+            SystemSchema.workflow_status,
+            SystemSchema.workflow_queue.c.workflow_uuid
+            == SystemSchema.workflow_status.c.workflow_uuid,
+        )
+
+        if input.get("queue_name"):
+            query = query.where(
+                SystemSchema.workflow_queue.c.queue_name == input["queue_name"]
+            )
+
+        if input.get("status"):
+            query = query.where(
+                SystemSchema.workflow_status.c.status == input["status"]
+            )
+        if "start_time" in input and input["start_time"] is not None:
+            query = query.where(
+                SystemSchema.workflow_status.c.created_at
+                >= datetime.datetime.fromisoformat(input["start_time"]).timestamp()
+                * 1000
+            )
+        if "end_time" in input and input["end_time"] is not None:
+            query = query.where(
+                SystemSchema.workflow_status.c.created_at
+                <= datetime.datetime.fromisoformat(input["end_time"]).timestamp() * 1000
+            )
+        if input.get("limit"):
+            query = query.limit(input["limit"])
```

Comment:
I think we should sort the query in ASC `created_at`, to provide some order for cases where there's more workflows in the requested time windows than the limit allows.

Reply:
Added sort, made it descending for consistency with the other commands